### PR TITLE
MCprep RC Scripts to Dev

### PR DESCRIPTION
**Pull request by StandingPadAnimations**
This moves the RC scripts to `dev` so we can make things easier

### CODE REVIEW

File: rc-files/download_resources.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Get the latest stable release and unzip
+wget "https://github.com/Moo-Ack-Productions/MCprep/releases/download/3.5.3/MCprep_addon_3.5.3.zip"
```
Comment (TheDuckCow) [inline rc-files/download_resources.sh:2]: Should either be a TODO or maybe even just a $1 input flag for tag version.

Could be fancier and use something like `git describe --tags --abbrev=0` to just get the latest tag and directly use that. The hard coding I just think will get overlooked is all

File: rc-files/HOW_TO_BUILD.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Step 1. Run `download_resources.sh` to add the necesary asset files
+
+Step 2. Apply all patches in the `patches` folder
```
Comment (TheDuckCow) [inline rc-files/HOW_TO_BUILD.md:3]: Could be nice to include the sample command for the one patch we do have (would be this right? `git apply rc-bl_info.patch`)

**Review by TheDuckCow — APPROVED** [review]
Approving with some minor suggestions, could do now or in a follow up TODO

File: rc-files/HOW_TO_BUILD.md (hunk 2 above)
Comment (StandingPadAnimations) [reply rc-files/HOW_TO_BUILD.md:3]: Yeah, although we actually have 2 patches (the other changes the tracker ID so we can better track RCs in the tracker)

**Comment by StandingPadAnimations** [comment]
Alright, merging now

**Pull request merged**